### ISSUE TITLE
[Limits, CSS] Add utility classes for common limit colors that don't set glyphs

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmct#1677]
A sub-task of #1558. Current limit CSS classes like `s-limit-red` define both icon and color, which create problems for developers seeking to use different icons. Modify limit classes such that colors are defined independently, with additional classes that set icons separately as well and inherit from the color classes. 

Devs should be able to apply something like `s-limit-red icon-arw-down` to an element and have that work.

**Comment by charlesh88** [comment]
@larkin Need a bit of clarity on how this would be used, as it gets complicated when dealing with tables... Can I assume that devs are in control of the HTML rendering and can place classes at will? So, they could attach a color class to a table row (if they wanted to) and an icon class to an individual cell of that row? Or put the color class and the icon class together on a cell?

**Comment by larkin** [comment]
@charlesh88 telemetry devs are not in control of the HTML, they can't choose where the classes go.  As it stands, limit classes are applied to the cell containing the value, never the row.  That's consistent across all our display types at this time.

**Comment by charlesh88** [comment]
@larkin Ok, but how are they getting the custom icon then? If they can't apply a CSS class of their choosing to a given bit of markup, and they don't want to write CSS that will override an existing class to specify a different icon, how will they do it?

**Comment by larkin** [comment]
@charlesh88 A telemetry adapter can apply a CSS class of their choosing (custom or from platform), but it cannot insert additional dom elements or otherwise change the structure of the view. The responsibilities are split:

**Telemetry adapter**: determines what css classes a given value should have.
**View**: determines where (what html element) to apply those css classes to.

**Comment by charlesh88** [comment]
@larkin Very cool, think I've got it now. So for a given telem value the adapter (per deployment) says "Hey view, apply `s-limit-red icon-sad-face` to telem element NUC_DNGR_OVRLOAD 1234". The view already knows where the limit CSS class goes and applies whatever it was given.

So I've restructured the CSS so that red and yellow limits by default don't specify an icon. So the adapter can specify whatever it wants, including icon classes, and the icon and color will display as desired. I added new red and yellow limit classes (`s-limit-red-icon`, `s-limit-yellow-icon`) that do specify the alert triangle icon. Upper and lower limits remain the same, specifying their icons, and must be used in concert with a limit color class.
